static/css/themes/kansha_flat/board.css

```diff
@@ -12,18 +12,18 @@
 }
 
 /* Board */
-.board .boardTitle {
+.board .board-title {
     color: #737373;
     font: normal 33px OpenSansLight;
 }
 
-.board .boardTitle a,
-.board .boardTitle a:hover {
+.board .board-title a,
+.board .board-title a:hover {
     color: #737373;
```

Comment:
I think  `#737373` must be the default text color, set on `body` and inherit everywhere